book/modules/mod-B/pages/shared

```diff
@@ -0,0 +1 @@
+../../shared/pages/shared
```

Comment:
Symlink too, just to make antora think that these files are really inside the consuming module (`mod-B`).